GAE/src/com/gallatinsystems/survey/SurveyHarness.java

```diff
@@ -29,7 +29,6 @@
     public static void main(String[] args) {
```

Comment:
Seems like this whole class can go as well.

Reply:
ok, removing